.jules/sentinel.md

```
@@ -0,0 +1,12 @@
+
+## 2024-05-04 - Mitigate XXE vulnerability in XML parsing
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)